packages/icon-button/README.md

```diff
@@ -0,0 +1,78 @@
+# React Icon Button
+
+A React version of an [MDC Icon Button](https://github.com/material-components/material-components-web/tree/master/packages/mdc-icon-button).
+
+## Installation
+
+```
+npm install @material/react-icon-button
+```
+
+## Usage
+
+```js
+import React from 'react';
+import IconButton from '@material/react-icon-button';
+import MaterialIcon from '@material/react-material-icon';
+
+class MyApp extends React.Component {
+  render() {
+    return (
+      <IconButton>
+        <MaterialIcon icon='favorite' />
+      </IconButton>
+    );
+  }
+}
+```
+
+You can use any other icon here such as Font Awesome, which is documented more in detail [here](https://github.com/material-components/material-components-web/tree/master/packages/mdc-icon-button#icon-button-toggle-with-font-awesome). If you're using [Google Font's Material Icons](https://material.io/tools/icons/?style=baseline), we recommend using our [Material Icon Component](../material-icon) as shown in the example above.
+
+## Variants
+
+### Icon Button Toggle
+
+If you need to use this component as an Icon Button Toggle, please read [this documentation](https://github.com/material-components/material-components-web/blob/v0.37.1/packages/mdc-icon-button/README.md#icon-button-toggle). The following is an example using data attributes to toggle between on/off states:
```

Comment:
Did you intend to lock this link to version 0.37.1?

Reply:
No, I think it should just be master or auto updated to whatever version it actually is. A lot of our readmes are out of sync in this section so we might want to just change them all to master to lower our maintenance.